example/transform/nougat_huggingface_QAs.ipynb

```diff
@@ -0,0 +1,807 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "b56ed8ef-bd07-4835-9693-a8c345183915",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "('## UNITED STATES SECURITIES AND EXCHANGE COMMISSION Washington, D.C. 20549\\n'\n",
+      " '**FORM 10-K**\\n'\n",
+      " '**(Mark One)**\\n'\n",
+      " '**(Exact name of registrant as specified in its charter)**\\n'\n",
+      " '**Delaware**\\n'\n",
+      " '**(State or other jurisdictions of**\\n'\n",
+      " '**(Interpretation of organizations)**\\n'\n",
+      " '**(I.R.S. Englurver**\\n'\n",
+      " '**(I.\\n'\n",
+      " '**AMAZON.COM, INC.**\\n'\n",
+      " '**FORM 10-K**\\n'\n",
+      " '**For the Fiscal Year Ended December 31, 2022**\\n'\n",
+      " '**INDEX**\\n'\n",
+      " '**PART I**\\n'\n",
+      " 'Item 1. Business\\n'\n",
+      " 'Item 1A. Risk Factors\\n'\n",
+      " 'Item 1B. Unresolved Staff Comments\\n'\n",
+      " 'Item 2. Properties\\n'\n",
+      " 'Item 3. Legal Proceedings\\n'\n",
+      " 'Item 4. Mine Safety Disclosures\\n'\n",
+      " '**PART II**\\n'\n",
+      " \"Item 5. Market for the Registrant's Common Stock, Related Shareholder \"\n",
+      " 'Matters, and Issuer Purchases of Equity Securities\\n'\n",
+      " 'Item 6. Reserved\\n'\n",
+      " \"Item 7. Management's Discussion and Analysis of Financial Condition and \"\n",
+      " 'Results of Operations\\n'\n",
+      " 'Item 7A. Quantitative ')\n"
+     ]
+    }
+   ],
+   "source": [
+    "pprint(extract_context[:800])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "429cf469-ed3e-45d9-9560-9403c8e3cfda",
+   "metadata": {},
+   "source": [
+    "Below are helper functions designed to process the output of `Nougat`, ensuring that the output context is efficiently processed by the Hugging Face model.\n",
+    "\n",
+    "#### Overview:\n",
+    "The `process_content` function is designed to process the content generated by `ExtractClient` above, particularly handling large sections and table content. It reads a markdown file, splits it into manageable sections, processes these sections to handle table content, and optionally utilizes OpenAI for further processing.\n",
+    "\n",
+    "#### Inputs:\n",
+    "- `content`: The string content generated by `ExtractClient`.\n",
+    "- `client_openAI`: An object representing the OpenAI client, used for processing sections of the markdown file.\n",
+    "\n",
+    "#### Workflow:\n",
+    "1. **Reading the File**: The function starts by reading the entire content of the markdown file specified by `file_path`.\n",
+    "2. **Initial Splitting**: The content is split into sections based on '##' headers. The first section is skipped if it's empty.\n",
+    "3. **Sub-Splitting for Large Sections**: Sections larger than a predefined word count (`max_word_count`) are further split using '###' headers.\n",
```

Comment:
qq: I thought Nougat ExtractPDFFlow already have a `split_op`. Why we need extract split here in this notebook.